library/ssl_tls.c

```diff
@@ -4121,22 +4088,36 @@ static uint16_t ssl_preset_default_sig_algs[] = {
     MBEDTLS_TLS1_3_SIG_ECDSA_SECP521R1_SHA512,
 #endif /* MBEDTLS_ECDSA_C && MBEDTLS_SHA384_C &&
           MBEDTLS_ECP_DP_SECP521R1_ENABLED */
+/*
```

Comment:
We should not change this list because of some limitations in the TLS 1.3 implementation (no version negotiation on the server side). I can see that if I remove those changes, in full config, tests where the server version is forced to TLS 1.3 and that use an RSA key fail. For those tests, I'd suggest to rather overwrite the default signature list (which is the TLS 1.2 one) and configure an RSS PSS signature algorithm.

Reply:
> Do you mean revert changes in ssl_preset_tls12_default_sig_algs and fix the issue with cmd line parameters?

Yes

Reply:
both cases are fixed. please review